### PR TITLE
Create Merchant Items Show Page

### PR DESCRIPTION
**Pull request by MadelineMauser**
- Adds the feature test for the Merchant Items show page
- Adds the Items controller
- Adds the view for the Merchant Items show page
- Adds the route to the Merchant Items show page
- All tests passing!

### CODE REVIEW

**Review by kenzjoy — APPROVED** [review]
Approved! Moving right along :)

File: app/views/items/show.html.erb [hunk 1]
```diff
@@ -0,0 +1,3 @@
+<p>Item: <%=@item.name%></p>
+<p>Description: <%=@item.description%></p>
+<p>Current price: <%=@item.unit_price%></p>
```
Comment (msakr21) [inline app/views/items/show.html.erb:3]: Could add a $ before/outside the erb portion here.

**Review by msakr21 — APPROVED** [review]

**Review by msakr21 — COMMENTED** [review]
Due to merging a PR this now has a conflict. If possible, please resolve locally and resubmit PR.

**Review by msakr21 — CHANGES_REQUESTED** [review]
Conflict needs resolution, preferably locally, before merging.

**Comment by MadelineMauser** [comment]
Refactors to remove conflict with main branch, adds remaining features described by Merchant Items Show Page user story.
-Refactors routes to use resources
-Merges the merchant_items controller and items controller
-Refactors tests to use route helpers
- Adds links to item names in merchant items index that redirect to that item's show page
-Changed merchant item show page view to display unit_price in dollars

**Review by willjw2 — APPROVED** [review]
Looks good, nice work changing the routes to use resources

**Review by msakr21 — APPROVED** [review]
Looks good!

**Pull request merged**